library/default-effects-extra.scd

```diff
@@ -0,0 +1,42 @@
+// Waveloss
+// Divides an audio stream into tiny segments, using the signal's
+// zero-crossings as segment boundaries, and discards a fraction of them.
+
+(
+SynthDef("waveloss" ++ ~dirt.numChannels, { |out, drop = 1|
+  var sig;
+
+  sig = In.ar(out, ~dirt.numChannels);
+  sig = WaveLoss.ar(sig, drop, outof: 100, mode: 2);
+  ReplaceOut.ar(out, sig)
+}).add;
+
+~dirt.addModule('waveloss', { |dirtEvent|
+  dirtEvent.sendSynth('waveloss' ++ ~dirt.numChannels,
+    [
+      drop: ~waveloss,
+      out: ~out
+    ]
+  )
+}, { true });
```

Comment:
should be:
`} , { ~waveloss.notNil }`

Otherwise, the module will be run on every synth.